spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/PropertiesProvider.scala

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.standardization
+
+import org.apache.spark.sql.{DataFrameReader, SparkSession}
+import org.slf4j.{Logger, LoggerFactory}
+import za.co.absa.enceladus.common._
+import za.co.absa.enceladus.dao.MenasDAO
+import za.co.absa.enceladus.model.Dataset
+import za.co.absa.enceladus.standardization.config.StandardizationConfig
+import za.co.absa.enceladus.utils.unicode.ParameterConversion._
+
+import scala.collection.immutable.HashMap
+
+class PropertiesProvider {
```

Comment:
Same thing.
It's also confusing having two classes of same name, just different path, similar function, but no relationship.

Reply:
They used to be called Readers and it seems that they can have the same name in different packages because they are only used in their own package

Reply:
I know it does work, but it's still confusing IMHO. 🤔